articles/quickstart/native/flutter/interactive.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Add login to your Flutter app
+default: true
+description: This tutorial demonstrates how to add user login with Auth0 to an Android or iOS Flutter application using the Auth0 Flutter SDK
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - flutter
+  - dart
+  - ios
+  - android
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/build
+  - files/main
+  - files/profile
```

Comment:
I think this needs to set the path in github, comparable to https://github.com/auth0/docs/blame/master/articles/quickstart/spa/angular/interactive.md#L11

But:
```
github:
  path: sample
```

Reply:
Thanks, fixed in 6f95a3085